client/SDL/SDL3/sdl_clip.cpp

```diff
@@ -671,27 +701,51 @@ const void* sdlClip::ClipDataCb(void* userdata, const char* mime_type, size_t* s
 	}
 
 	{
-		HANDLE hdl[2] = { clip->_event, freerdp_abort_event(clip->_sdl->context()) };
-		auto status = WaitForMultipleObjects(ARRAYSIZE(hdl), hdl, FALSE, INFINITE);
-		if (status != WAIT_OBJECT_0)
-			return nullptr;
+		HANDLE hdl[2] = { freerdp_abort_event(clip->_sdl->context()), clip->_event };
```

Comment:
we should drop `cppcoreguidelines-avoid-c-arrays`, there's no rational to use complicated `C++` construct that could not behave as expected instead of plan C arrays, especially when it's for calling a `C` function.

Reply:
yes, that one we can agree on.